README.fr.md

```diff
@@ -0,0 +1,122 @@
+
+
+### Installation
+
+* Clonez ce projet, par exemple:
+  `git clone git@github.com: osmlab / nom-suggestion-index.git»
+* `cd` dans le dossier du projet,
+* Exécutez `npm install` pour installer les bibliothèques
+### À propos de l'index
+#### Fichiers générés (ne pas modifier):
+Fichiers prédéfinis (utilisés par les éditeurs OSM):
+* `dist / name-suggestions.json` - Préréglages de suggestion de nom
+* `dist / name-suggestions.min.json` - Préréglages de suggestion de nom, minifiés
+* `dist / name-suggestions.presets.xml` - Préréglages de suggestion de nom, comme XML prédéfini de style JOSM
+Listes de noms:
+* `dist / names_all.json` - tous les noms et balises fréquents collectés depuis OpenStreetMap
+* `dist / names_discard.json` - sous-ensemble de` names_all` que nous éliminons
+* `dist / names_keep.json` - sous-ensemble de` names_all` que nous conservons
+* `dist / wikidata.json` - données de marque en cache récupérées à partir de Wikidata
+#### Fichiers de configuration (modifiez-les):
+* `config / *`
+  * `config / filters.json` - Expressions régulières utilisées pour filtrer` names_all` dans `names_keep` /` names_discard`
+* `marques / *` - Fichiers de configuration pour chaque type d'entreprise de marque, organisés par balise OpenStreetMap
+  * `marques / équipement / *. json`
+  * `marques / loisirs / *. json`
+  * `marques / boutique / *. json`
+  * `marques / tourisme / *. json`
+  * `marques / bureau / *. json`
+
+: point_right: Voir [CONTRIBUTING.md](CONTRIBUTING.md) pour savoir comment contribuer à cet index.
+
+
+### Création de l'index
+
+* `npm run build`
+  * Régénère `dist / names_keep.json` et` dist / names_discard.json`
+  * Toutes les nouvelles entrées de `names_keep` qui ne sont pas déjà présentes dans l'index y seront ajoutées
+  * Génère de nombreux avertissements pour suggérer des mises à jour de `marques / ** / *. Json`
+
+
+### Autres commandes
+
+* `npm run wikidata` - Récupère les données utiles de Wikidata - étiquettes, descriptions, logos, etc.
+* `npm run` - Liste les autres outils disponibles
+
+### Mise à jour de `dist / names_all.json` depuis la planète
+
+Cela prend beaucoup de temps et beaucoup d'espace disque. Cela peut être fait occasionnellement par les responsables du projet.
+Vous n'avez pas besoin de suivre ces étapes pour contribuer à l'index.
+
+- Installer l'outil de ligne de commande `osmium` et le package de nœuds (peut être disponible uniquement dans certains environnements)
+  - `apt-get install osmium-tool` ou` brew install osmium-tool` ou similaire
+  - `npm install --no-save osmium`
+- [Télécharger la planète](http://planet.osm.org/pbf/)
+  - `curl -L -o planet-latest.osm.pbf https: // planet.openstreetmap.org / pbf / planet-latest.osm.pbf»
+- Préfiltrez le fichier planète pour inclure uniquement les éléments nommés avec les clés que nous recherchons:
+  - `balises osmium-filtre planet-latest.osm.pbf -R nom -o nommé.osm.pbf`
```

Comment:
looks like the osmium commands got translated by whatever program you used? 🤔

Reply:
Sorry, translating in the middle of the night is tiring, kinda ignored the markup, just spaced everything out and translate word-by-word. Sorry about that... 😭😭😭

Reply:
Sorry about that, I was tired and translated roughly half, read the translation over but missed the code, etc... 😢😴